### DISCUSSION TITLE
How to set the database path using Entity Framework Core?

### DISCUSSION THREAD

**Discussion by haijialiu** [microsoft/microsoft-ui-xaml#10093] [Q&A] [answered]
I want to use sqlite with EF core in packaged winui 3 project, how to set Data Source?  The Relative path "Data Source=Data.db" is not work, the absolute path is ok.

**Comment by ghost1372** [comment, marked as answer]
If you want to use db next to exe you can use this code: (modify path for assets\DataBase)

```
var dbFile = @$"{AppContext.BaseDirectory}\Assets\DataBase\Al-Anvar.db";
        optionsBuilder.UseSqlite($"Data Source={dbFile}");
    
```

**Reply by haijialiu** [reply]
It's useful to me, thank you. :)
